conf/config.toml

```diff
@@ -40,4 +40,5 @@ max-balance-count = 16
 max-balance-retry-per-loop = 10
 max-balance-count-per-loop = 3
 max-transfer-wait-count = 3
-max-store-down-duration = 60
+max-peer-down-duration = "30m"
```

Comment:
I think we should discuss the max-peer-down-duration variable name, it is confused.